### PR TITLE
Add two new books: React and Scratch

### PR DESCRIPTION
**Pull request by rosepac**
## What does this PR do?
Add Resource(s)

### Description 
Book about react and book about sractch.

### Why is this valuable (or not)
Because it is one of the best react books translated into Spanish.

### How do we know it's really free?
Check it.

### For book lists, is it a book?
Yes. 

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-books-es.md [hunk 1]
```diff
@@ -241,6 +241,7 @@
 #### React
 
 * [Desarrollo de Aplicaciones Web con React.js y Redux.js](https://leanpub.com/react-redux) (PDF, EPUB, MOBI)
+* [El Camino para Aprender React](https://leanpub.com/the-road-to-learn-react-spanish) (PDF, EPUB, MOBI)
```
Comment (eshellman) [inline free-programming-books-es.md:244]: On the English list Leanpub books using this model have an added notation "(Leanpub account or valid email requested)". Could you add  a spanish version of this to your addition? (Separately, we can add the same notation to other Leanpub books on the spanish list.)

Also, it helps to identify books, especially translated works, to add the author name to the listing.

File: free-programming-books-es.md [hunk 2]
```diff
@@ -351,6 +352,7 @@
 ### Scratch
 
 * [Informática Creativa](https://github.com/programamos/GuiaScratch) (PDF)
+* [Manual de Scratch 2](https://lsi.vc.ehu.es/pablogn/docencia/FdI/Scratch/Aprenda%20a%20programar%20con%20Scratch%20en%20un%20par%20de%20tardes.pdf) (PDF)
```
Comment (eshellman) [inline free-programming-books-es.md:355]: Good resource, but the site does not have a valid certificate for https. Use http instead.

**Comment by eshellman** [comment]
please resolve conflicts.

**Comment by rosepac** [comment]
> please resolve conflicts.

Because they have erased books .. As it? -> El camino para aprender React (https://leanpub.com/the-road-to-learn-react-spanish) ¿EM? .. 🤦‍♂️

**Pull request merged**